misk/src/main/kotlin/misk/tracing/TracingModule.kt

```diff
@@ -1,31 +0,0 @@
-package misk.tracing
-
-import com.google.common.util.concurrent.Service
-import misk.inject.KAbstractModule
-import misk.logging.getLogger
-import misk.tracing.backends.TracingBackendConfig
-import misk.tracing.backends.jaeger.JaegerBackendModule
-import misk.tracing.backends.noop.NoopTracerBackendModule
-import misk.tracing.backends.zipkin.ZipkinBackendModule
-import kotlin.reflect.full.memberProperties
-
-private val logger = getLogger<TracingModule>()
-
-class TracingModule(val config: TracingConfig) : KAbstractModule() {
```

Comment:
Removed without replacement. I’m going to try to circle back to this.
https://github.com/square/misk/issues/308